css/common.css

```diff
@@ -2100,3 +2100,38 @@ input::-webkit-inner-spin-button {
     -webkit-appearance: none;
     margin: 0;
 }
+
+.drop-citation-btn {
+    color: white;
+    font-size: 13px;
+    border: none;
+    cursor: pointer;
+    background-color: #099242;
+    padding: 0.1px 3px 1px 3px;
+    border-radius: 4px 4px 4px 4px;
+    display: inline-block;
+    position: relative;
+    right: -10px;
+}
+.drop-citation-btn:focus {
+    outline: none;
+}
+.citation-content {
+    display: none;
+    position: absolute;
+    left: 45%;
+    background-color: white;
+    min-width: 120px;
+    overflow: auto;
+    border:  1px solid darkseagreen;
+    border-radius: 4px 4px 4px 4px;
+    z-index: 1; /*stack in front */
+}
+.citation-content a {
+    color: black;
+    padding: 8px 14px;
+    text-decoration: none;
+    display: block;
+    font-size: 12px;
+}
+.citation-content a:hover {background-color: #F3FAF6;}

```

Comment:
These new CSS rules need to be moved into a new or existing `less` file in one of the sub-directories in `less/current` when you have finished development. If a new less file is used then it needs to be defined in `current.less` for import into `common.css`.

Reply:
These CSS rules has been moved to `/current/pages/dataset.less`.

Reply:
This CSS code should be removed from `common.css` which should be what it was in `develop` branch.

Reply:
I rebuilt the containers and the CSS is working now.